.gitignore

```diff
@@ -21,3 +20,4 @@ config/webpack/paths.json
 cypress/screenshots
 cypress/videos
 .DS_Store
+.yarn/
```

Comment:
This is because we always end up with a bonus file from the master branch when switching branches to morphy and it shouldn't be committed.